### PR TITLE
Update readme to specify hard coded values

### PR DESCRIPTION
**Pull request by faizanahmad055**

### CODE REVIEW

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. ``docker pull stakater/gitwebhookproxy:PR-36-1``

File: README.md [hunk 1]
```diff
@@ -63,23 +65,44 @@ Update the above mentioned parameters according to your configuration and update
...
+  secret: ZHVtbXlzZWNyZXQ=
+```
+
+3. Change below mentioned values in `ConfigMap` in `gitwebhookproxy.yaml`
+
 ```yaml
-upstreamURL: https://jenkins.tools.stackator.com
+data:
+  provider: github
+  upstreamURL: https://jenkins.tools.stackator.com
```
Comment (waseem-h) [inline README.md]: Remove our URL

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. ``docker pull stakater/gitwebhookproxy:PR-36-2``

File: README.md [hunk 2]
```diff
@@ -50,36 +50,59 @@ For Vanilla manifests, you can either first clone the respository or download th
...
 
 *Note:* Make sure to update the `port` in deployment.yaml as well as service.yaml if you change the default `listenAddress` port.
 
 ### Helm Charts
 
-Alternatively if you have configured helm on your cluster, you can deploy the controller via helm chart located under `deployment/kubernetes/chart` folder.
+Alternatively if you have configured helm on your cluster, you can add gitwebhookproxy to helm from our public chart repository and deploy it via helm using below mentioned commands
+
+```bash
+helm repo add stakater https://stakater.github.io/stakater-charts
```
Comment (waseem-h) [inline README.md]: See how IMC has this written in docs. It also shows how to customize the chart before applying @faizanahmad055

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. ``docker pull stakater/gitwebhookproxy:PR-36-3``

**Pull request merged**